tests/ignite/handlers/test_lr_finder.py

```diff
@@ -274,6 +274,12 @@ def test_lr_policy(lr_finder, to_save, dummy_engine, dataloader):
     lr = lr_finder.get_results()["lr"]
     assert all([lr[i - 1] < lr[i] for i in range(1, len(lr))])
 
+def test_multi_group_optimizer(lr_finder, dummy_engine_mulitple_param_groups, to_save_mulitple_param_groups, dataloader):
+    with lr_finder.attach(dummy_engine_mulitple_param_groups, to_save_mulitple_param_groups, start_lr=[0.1, 0.1, 0.01], end_lr=[1.0, 1.0, 0.1]) as trainer_with_lr_finder:
```

Comment:
Since `lr_finder`'s `step_mode` accepts two values (`exp` and `linear`), we should test both of them. So let's parametrize the test:

```suggestion
@pytest.mark.parametrize("step_mode", ["exp", "linear"])
def test_multi_group_optimizer(lr_finder, dummy_engine_mulitple_param_groups, to_save_mulitple_param_groups, dataloader, step_mode):
    with lr_finder.attach(dummy_engine_mulitple_param_groups, to_save_mulitple_param_groups, start_lr=[0.1, 0.1, 0.01], end_lr=[1.0, 1.0, 0.1], step_mode=step_mode) as trainer_with_lr_finder:
```
This way [pytest](http://pytest.org/) runs two tests, one for each `step_mode`.